website/static/js/addProjectPlugin.js

```diff
@@ -474,4 +482,34 @@ var Select2Template = {
     }
 };
 
+var SelectStorageLocation = {
+    view: function(ctrl, options) {
+        return m('select.p-t-sm', {config: SelectStorageLocation.config(options)},
+            [
+            window.contextVars.storageRegions.map(function(region) {
```

Comment:
Any reason to pull directly from `window.contextVars` instead of defining a prop in the controller? 
```
self.storageLocations = m.prop(window.contextVars.storageRegions)
```
(same question on LL503)